webui/react/src/routes/utils.ts

```diff
@@ -0,0 +1,117 @@
+import { pathToRegexp } from 'path-to-regexp';
+import { MouseEvent, MouseEventHandler } from 'react';
+
+import history from 'routes/history';
+import { Command, CommandType } from 'types';
+import { clone } from 'utils/data';
+
+import { appRoutes } from './routes';
+
+export const serverAddress = (path: string): string => {
```

Comment:
blocking: the returned value is no longer just the server address. thoughts on making the path either optional or leaving it out as all it does is:  add it to the serverAddress at all times

Reply:
made `path` optional and kept the logic mostly the same as before.